src/app/core/pipes/orderByName.pipe.ts

```diff
@@ -0,0 +1,12 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import _ from 'lodash';
+
+@Pipe({
+  name: 'myOrderByName'
+})
+
+export class OrderByNamePipe implements PipeTransform {
```

Comment:
Ten Pipe służy do filtrowania, czyli powinien się na przykład nazywać `FilterByName`. To jest bardzo mylące w czytaniu kodu...